lib/workers/repository/updates/generate.spec.ts

```diff
@@ -15,7 +15,7 @@ beforeEach(() => {
 describe('workers/repository/updates/generate', () => {
   describe('generateBranchConfig()', () => {
     it('does not group single upgrade', () => {
-      const branch: BranchUpgradeConfig[] = [
+      const branch: BranchUpgradeConfig[] = partial<BranchUpgradeConfig>([
```

Comment:
```suggestion
      const branch = partial<BranchUpgradeConfig>([
```


please do same for others below

Reply:
Could you comment on the [TBD] change suggestion in the PR description